sentry-spring/src/main/java/io/sentry/spring/graphql/SentryDgsSubscriptionHandler.java

```diff
@@ -0,0 +1,29 @@
+package io.sentry.spring.graphql;
+
+import graphql.execution.instrumentation.parameters.InstrumentationFieldFetchParameters;
+import io.sentry.IHub;
+import io.sentry.graphql.ExceptionReporter;
+import io.sentry.graphql.SentrySubscriptionHandler;
+import org.jetbrains.annotations.NotNull;
+import reactor.core.publisher.Flux;
+
+public final class SentryDgsSubscriptionHandler implements SentrySubscriptionHandler {
```

Comment:
put this here as it requires `reactor`. If we added reactor to `sentry-graphql` we could rename it to something generic and put it there. Not sure we should do that tho.